src/lua/luaTask.c

```diff
@@ -227,9 +275,16 @@ static void lua_task(void *params)
         };
 
         int consecutive_failures = 0;
-        for(portTickType xLastWakeTime;;
-            vTaskDelayUntil(&xLastWakeTime, state.callback_interval)) {
-                xLastWakeTime = xTaskGetTickCount();
+	portTickType wake_tick = 0;
+        for(;;) {
```

Comment:
so.. how does this play into how responsive the lua command is based on the current tickRate()?

Will a user enter a command, and then wait for the current tick to fire to see it execute? e.g. default tick rate is 1Hz - it could take up to 1 second for a command to be processed?